packages/analytics-types/tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src/**/*"],
+  "include": ["src/**/*", "test/**/*"],
   "compilerOptions": {
-    "module": "commonjs",
-    "outDir": "lib",
-    "types": [],
```

Comment:
is it expected to remove this `"types": [],` which disables the eager loading of all `@types` packages that are installed ?

Reply:
Good catch. We should keep it. Will open another PR.

Reply:
https://github.com/amplitude/Amplitude-TypeScript/pull/226